pages/articles/[slug].tsx

```diff
@@ -28,13 +28,10 @@ const ArticlePage: NextPage = ({
         />
         <meta key="description" property="description" content={post.excerpt} />
         <meta property="og:type" content="article" />
-        <meta
-          property="og:url"
-          content={`https://codu.co/articles/${post.slug}`}
-        />
+        <meta property="og:url" content={`${host}/articles/${post.slug}`} />
         <meta
           property="og:image"
-          content={`${process.env.BASE_URL || `http://localhost:3000`}/api/og?title=${post.title}`}
+          content={`${host}/api/og?title=${post.title}`}
```

Comment:
Love it! 💪 Wondering if there is some kind of win where this could even be defined in the `Layout` 🤔 in a blanket style using page title and an override where defined.

Reply:
Just wondering would we have to server side render everything then. 🤔 Or were you thinking something different?